app/controllers/catalog_controller.rb

```diff
@@ -34,10 +34,17 @@ class CatalogController < ApplicationController
     config.add_show_field 'watermark_tesim', label: 'Watermark'
     config.add_show_field 'colophon_tesim', label: 'Colophon'
 
+    config.add_facet_field 'annotation_tags_ssim', label: 'Tags', sort: 'index', partial: 'blacklight/hierarchy/facet_hierarchy'
+    config.facet_display = {
+      hierarchy: {
+        'annotation_tags' => [['ssim'], '|']
```

Comment:
Is the hierarchy a `|` or a `-` for our case? (or maybe we don't know yet because we haven't indexed any)

Reply:
Yeah, I'm not sure we know. I figured `|` is a less likely character to appear naturally, so it seemed like a good first choice.